controllers/office.js

```diff
@@ -1,46 +1,27 @@
 const OfficeModel = require('../models/office')
 
-module.exports.get = function get (req, res) {
-  OfficeModel.find({}).exec(function (err, value) {
-    if (err) return res.status(500).send()
-    res.json({value})
-  })
+module.exports.get = async function get (req, res) {
+  const data = await OfficeModel.find({})
+  res.json({data})
 }
 
-module.exports.getOne = function getOne (req, res) {
-  OfficeModel.findOne({ _id: req.params.id }, req.body)
-  .exec(function (err, value) {
-    if (err) return res.status(500).send()
-    res.json({value})
-  })
+module.exports.getOne = async function getOne (req, res) {
+  const data = await OfficeModel.findOne({ _id: req.params.id }, req.body)
```

Comment:
why are you sending req.body to findOne? take a look at mongoose documentation, it does not require a second parameter when you use await